dragonchain/broadcast_processor/broadcast_functions.py

```diff
@@ -102,13 +106,30 @@ def increment_storage_error_sync(block_id: str, current_level: int) -> None:
 
 async def get_blocks_to_process_for_broadcast_async() -> List[Tuple[str, int]]:
     """Get the blocks scheduled to be checked by the broadcast processor right now
-    Args:
     Returns:
         List of (block_id, score) tuples to be processed by the broadcast processor (limit of 1000)
     """
     return await redis.z_range_by_score_async(IN_FLIGHT_KEY, 0, int(time.time()), withscores=True, offset=0, count=1000)
 
 
+async def get_notification_verifications_for_broadcast_async() -> set:
```

Comment:
add tests for all the new functions in this file

Reply:
done